space/internal/components/migration/runner.go

```diff
@@ -0,0 +1,116 @@
+package migration
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/anyproto/any-sync/app"
+	"github.com/anyproto/any-sync/app/logger"
+	"github.com/hashicorp/go-multierror"
+	"go.uber.org/zap"
+
+	"github.com/anyproto/anytype-heart/pkg/lib/localstore/objectstore"
+	"github.com/anyproto/anytype-heart/space/clientspace"
+	"github.com/anyproto/anytype-heart/space/internal/components/migration/common"
+	"github.com/anyproto/anytype-heart/space/internal/components/migration/readonlyfixer"
+	"github.com/anyproto/anytype-heart/space/internal/components/migration/systemobjectreviser"
+	"github.com/anyproto/anytype-heart/space/internal/components/spaceloader"
+)
+
+const (
+	CName     = "common.components.migration-runner"
```

Comment:
lets name the component with the path (e.g. without - runner)

Reply:
Thank you! Changed the name